### DISCUSSION TITLE
Name to use for a package

### DISCUSSION THREAD

**Discussion by lahwaacz** [ginkgo-project/ginkgo#1095] [Q&A] [answered]
I would like to create a Ginkgo package for Arch Linux in the [AUR](https://aur.archlinux.org/). However, the [ginkgo-git](https://aur.archlinux.org/packages/ginkgo-git) name is already taken. (The "-git" suffix is a convention required for packages built from git repositories.) Can you recommend a different name to use for the package?

**Comment by pratikvn** [comment, marked as answer]
Sorry for the late reply. If you are still looking for a name, I can recommend `ginkgo-lib-git` or `ginkgo-project-git`.

**Reply by upsj** [reply]
Claudius and I discussed `ginkgo-hpc-git` as an alternative, since he was considering creating a nixOS package

**Reply by lahwaacz** [reply]
Thanks, I'll go with `ginkgo-hpc-git` since `lib` and `project` seem too generic.

**Reply by lahwaacz** [reply]
I've created the package: https://aur.archlinux.org/packages/ginkgo-hpc-git
